services/ai-navigator-app/0.2.0/helmrelease.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: ai-navigator-cluster-info-api
```

```suggestion
  name: ai-navigator-cluster-info-api-helmrelease
```
the name ll collide with kustomization created by kommander controllers.

---

Thank you!